src/diagnostic.c

```diff
@@ -66,227 +66,228 @@ struct diagnostic_data {
  * PM_WARNING_VERBOSE_NOT_NIL if they appear for `ruby -c -e 'code'`,
  * or PM_WARNING_VERBOSE_TRUE if they appear only with -w: `ruby -w -c -e 'code'`.
```

Comment:
```suggestion
 * PM_WARNING_LEVEL_DEFAULT if they appear for `ruby -c -e 'code'`,
 * or PM_WARNING_LEVEL_VERBOSE if they appear only with -w: `ruby -w -c -e 'code'`.
```

Reply:
Also there are a few mentions of `pm_diagnostic_level_t` left, but that doesn't exist anymore.